vars/piperPipelineStageInit.groovy

```diff
@@ -23,6 +23,10 @@ import static com.sap.piper.Prerequisites.checkScript
      * Defines the library resource containing the container map.
      */
     'containerMapResource',
+    /**
+     * Provides a clone from the specified repository.
```

Comment:
As it will apprear in the official documentation. Just from the sentence I would not know what this parameter is about and how I can use it or what the content has to be

Reply:
How about now?